yaml/csi-driver/edge/hpe-csi-k8s-1.22.yaml

```diff
@@ -0,0 +1,965 @@
+# Configuration to deploy the HPE CSI driver compatible with
+# Kubernetes = v1.21
```

Comment:
@datamattsson Should this be v1.22 here?

Reply:
Thanks! Fixed!